src/components/sync-connected-sites.tsx

```diff
@@ -272,6 +272,9 @@ const SyncConnectedSitesList = ( {
 								{ connectedSite.environmentType === 'production' && (
 									<EnvironmentBadge type="production" />
 								) }
+								{ connectedSite.environmentType === 'sandbox' && (
+									<EnvironmentBadge type="sandbox" />
+								) }
```

Comment:
I've been investigating if we could pass the environment type directly to the badge component, and  optimized it like: 
`<EnvironmentBadge type={ connectedSite.environmentType } />`

But I think it's better to leave it as it is for readability reasons. If we have appetite in the future to optimize it , we could set the EnvironmentType after reading the data from the API on `src/hooks/use-fetch-wpcom-sites/index.tsx`.

Reply:
> I've been investigating if we could pass the environment type directly to the badge component, and optimized it like:
<EnvironmentBadge type={ connectedSite.environmentType } />

Yes, I was trying making similar changes when working on the PR, but didn't like the outcomes. 😄 

> [...] If we have appetite in the future to optimize it , we could set the EnvironmentType after reading the data from the API on src/hooks/use-fetch-wpcom-sites/index.tsx.

This is something I haven't considered. Thanks!